app/views/admin/tracks/index.html.haml

```diff
@@ -6,59 +7,64 @@
-              %span.label{style: "background-color: #{track.color}; color: #{ contrast_color(track.color) }"}
-                = track.color
+              = track.start_date.strftime('%A, %B %-d. %Y') if track.start_date
             %td
-              - if track.self_organized?
-                = track.state
-              - else
-                N/A
+              = track.end_date.strftime('%A, %B %-d. %Y') if track.end_date
             %td
+              = link_to track.submitter.name, admin_user_path(track.submitter) if track.self_organized?
+            %td.text-center{ 'id' => "cfp_switch_#{track.id}", 'data-order' => track.cfp_active.to_s }
+              = check_box_tag "#{@conference.short_title}_#{track.short_name}", track.id, track.cfp_active,
+                class: 'switch-checkbox', method: :patch,
+                url: toggle_cfp_inclusion_admin_conference_program_track_path(@conference.short_title, id: track.short_name)+"?included=",
+                data: { size: 'small',
+                        on_color: 'success',
+                        off_color: 'warning',
+                        on_text: 'Yes',
+                        off_text: 'No' }
+
+            %td.text-center
               - if track.self_organized?
-                = check_box_tag "#{@conference.short_title}_#{track.short_name}", track.id, track.cfp_active,
-                  class: 'switch-checkbox', method: :patch,
-                  url: toggle_cfp_inclusion_admin_conference_program_track_path(@conference.short_title, id: track.short_name)+"?included=",
-                  data: { size: 'small',
-                          on_color: 'success',
-                          off_color: 'warning',
-                          on_text: 'Yes',
-                          off_text: 'No' }
+                .btn-group
+                  %button{ type: 'button', class: 'btn btn-link dropdown-toggle', 'data-toggle' => 'dropdown' }
+                    = track.state.humanize
+                    %span.caret
+                  %ul.dropdown-menu{ role: 'menu' }
+                    = render 'change_state_dropdown', track: track
               - else
-                %i.fa.fa-check
+                = track.state.humanize
             %td
               .btn-group{role: "group"}
-                = link_to 'Edit', edit_admin_conference_program_track_path(@conference.short_title, track),
-                method: :get, class: 'btn btn-primary'
+                - if can? :edit, track
+                  = link_to 'Edit', edit_admin_conference_program_track_path(@conference.short_title, track), class: 'btn btn-primary'
                 - if can? :destroy, track
-                  = link_to 'Delete', admin_conference_program_track_path(@conference.short_title, track),
-                  method: :delete, class: 'btn btn-danger',
+                  = link_to 'Delete', admin_conference_program_track_path(@conference.short_title, track), method: :delete, class: 'btn btn-danger',
```

Comment:
When is it possible to delete a track? 
It could be when it is not self_organized, to keep things the same as before for conferences that don't have a cfp for tracks at all.